python/lsst/afw/cameraGeom/assembleImage.py

```diff
@@ -55,10 +55,15 @@ def _insertPixelChunk(outView, inView, amplifier, hasArrays):
 def assembleAmplifierImage(destImage, rawImage, amplifier):
     """!Assemble the amplifier region of an image from a raw image
 
-    @param[in,out] destImage  assembled image (lsst.afw.image.Image or MaskedImage);
+    Parameters
+    ----------
+    destImage : `Unknown`
+        assembled image (lsst.afw.image.Image or MaskedImage);
```

Comment:
> nor are there classes such as `lsst.afw.image.Image` in Python

Actually, there now are such classes, since the pybind11 conversion - they're abstract base classes for the wrapped template instantiations with suffixes, and I think it's probably quite reasonable to link to them in docs when any type of image can be passed (they don't have particularly useful docs themselves, but neither does anything that originates in C++ right now).  The same is true of `MaskedImage` and `Exposure`.